src/components/EventsForm.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  Alert,
+  Button,
+  DatePicker,
+  Form,
+  Input,
+  InputNumber,
+  Upload,
+} from 'antd';
+import { Moment } from 'moment';
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import styled from 'styled-components';
+import { UserAuthenticationReducerState } from '../auth/ducks/types';
+import {
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+} from '../utils/asyncRequest';
+import { FileField } from '../utils/fileEncoding';
+import FormContainer from './FormContainer';
+
+const { Dragger } = Upload;
+const { TextArea } = Input;
+
+const PaddedAlert = styled(Alert)`
+  margin-top: 1em;
+  margin-bottom: 2em;
+`;
+
+export interface EventsFormData {
+  title: string;
+  capacity: number;
+  thumbnail?: FileField;
+  price: number;
+  description: string;
+  location: string;
+  start: Date;
+  end: Date;
+}
+
+export interface EventsFormInitialValues {
```

Comment:
⛏️ Same here, it would be nice but not necessary to have a base type to abstract common fields